utopia-remix/app/routes/projects.tsx

```diff
@@ -720,7 +730,7 @@ const ProjectCard = React.memo(
                 position: 'relative',
                 filter: project.deleted === true ? 'grayscale(1)' : undefined,
               }}
-              onMouseDown={onSelect}
```

Comment:
does it mean calling `onSelect()` somehow leads to a de-select?

Reply:
Yes, it's actually toggling the selection (maybe should've been called `toggleSelection`)

Reply:
sorry I missed this! could we rename this to toggleSelection then?